pkg/create/templates/test.go

```diff
@@ -106,7 +114,7 @@ test_connection() {
   local result=1
   while [ $attempt -le $max_attempts ]; do
     echo "Sending GET request to http://$(container_ip):${test_port}/"
-    response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    response_code=$(docker run --rm -it sequenceiq/alpine-curl curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
```

Comment:
yes it is, but on osx if using boot2docker, the container IP is not directly accessible. thus the use of an container, to be in the same network.